content/en/about/contribute/remove-content/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Remove Retired Documentation
+description: Details how to contribute retired documentation to Istio.
+weight: 4
+keywords: [contribute]
+---
+
+To remove documentation from Istio, please follow these simple steps:
+
+1. Remove the page.
+1. Reconcile the broken links.
+1. Submit your contribution to GitHub.
+
+## Remove the page
+
+Use `git rm -rf` to remove the directory containing the page.
+
+## Reconcile broken links
+
+To reconcile broken links, use this flowchart:
+
+{{< image width="40%"
```

Comment:
Enlarge diagram - to small to read in preview.

Reply:
set at 100% - will preview with our infra and resolve if it looks good.

Reply:
diagram looks to be correct size now (100%).